collector/src/main/java/com/bitmovin/analytics/data/Backend.kt

```diff
@@ -0,0 +1,8 @@
+package com.bitmovin.analytics.data
+
+import okhttp3.Callback
```

Comment:
Would it be easy to decouple this interface from `okhttp3`?

Reply:
Added `RetryCallback` interface that inherits this `okhttp3.Callback` interface